core/src/main/scala/org/broadinstitute/dsde/rawls/model/ResourceBufferModel.scala

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.dsde.rawls.model
+
+case class ProjectPoolId(value: String)
```

Comment:
RBS doesn't actually have a concept of ProjectPoolId, so I separated them so that we can ensure that we aren't pulling from the wrong type of pool. Since Rawls might not be pulling any other types of resources from RBS, this might not be super useful

Reply:
In that case, I'd vote to remove `PoolId`. If we ever start using RBS for other resources, we could add distinct `{ResourceName}PoolId` case classes then if we want to ensure that we aren't pulling from the wrong type of pool. At the moment, this feels like an unnecessary distinction to me

Reply:
I am going to second this, this smells a bit to me of premature optimization.